src/core/Akka.Streams/Implementation/IO/FileSubscriber.cs

```diff
@@ -76,6 +82,9 @@ protected override void PreStart()
             try
             {
                 _chan = _f.Open(_fileMode, FileAccess.Write);
+                if (_startPosition > 0) {
```

Comment:
either no curly braces or in the next line, but no java style :stuck_out_tongue: